homework/calculate/calculate.hpp

```diff
@@ -2,6 +2,23 @@
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+    
+    std::string result{};
+
+    if(command == "add"){
+        result = std::to_string(first + second);
+    }else if(command == "multiply"){
+        result = std::to_string(first * second);
+    }else if(command == "divide"){
+        if(second != 0){
+        result = std::to_string(first / second);
+        }else{
+        result = "Division by 0";
+        }
+    }else if(command == "subtract"){
+        result = std::to_string(first - second);
+    }else{
+        result = "Invalid data";
+    }
```

Comment:
Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..

When you use return inside `if` you don't need `if else` or `else`. Just write `if (sth) { return blabla; }`